### PR TITLE
refactor: remove redundant CSS declarations

### PR DESCRIPTION
**Pull request by lex111**
It's unnecessary code from which we can easily remove, and nothing breaks.

### CODE REVIEW

File: packages/core/styles/components/close.pcss [hunk 1]
```diff
@@ -6,10 +6,7 @@
  */
 
 .close {
```
Comment (slorber) [inline packages/core/styles/components/close.pcss:8]: is this close class still used anywhere?

**Review by slorber — APPROVED** [review]

File: packages/core/styles/components/close.pcss (hunk 1 above)
Comment (lex111) [reply packages/core/styles/components/close.pcss:8]: No, `close` class used only in context of alerts.

Comment (slorber) [reply packages/core/styles/components/close.pcss:8]: I see, so it's only used in Infima and not Docusaurus? 
So should we remove this?

Related the other issue, and in my opinion, we should only keep in Infima what we use in Docusaurus theme classic, but it really depends on what we want Infima to be in the end.

@yangshun what do you think?

Comment (lex111) [reply packages/core/styles/components/close.pcss:8]: This class is used by the PWA plugin:

image

Comment (slorber) [reply packages/core/styles/components/close.pcss:8]: I see thanks 👍 let's keep it then 🤪

**Pull request merged**